index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="CSS/style.css">
+    <title>Abiy Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <ul>
+            <li><a href="#"><img src="Images\MyLogo.svg" alt="" width="76px" height="20px"></a></li>
+            <li>
+               <nav><img src="Images\menu.svg" alt="" width="18px" height="12px"></nav>
+            </li>
+         </ul>
+    </header>
+    <!-- <img src="../Images/my_background.svg" alt="my background image"> -->
+    <section class="mobile-landing">
+        <h1> <span class="wrapper">I'm Mariane</span> <span class="wrapper">Glad to see</span> <span class="wrapper">you!</span></h1>
+        <p>
+            <span class="p-wrapper">I'm a software developer! I can help you</span> 
+            <span class="p-wrapper">build a product , feature or website Look</span> 
+            <span class="p-wrapper">through some of my work and</span> <span></span> 
+            <span class="p-wrapper">experience! If you like what you see and</span> 
+            <span class="p-wrapper">have a project you need coded, don't</span>
+            <span class="p-wrapper">hestiate to contact me.</span>
```

Comment:
- [ Required ] I think it would be better to use only the `p` tag to display this part of your portfolio,  semantically the `p` tag is the ideal tag for displaying text that spans multiple lines. Please rectify this.